src/app_service/service/community_tokens/service.nim

```diff
@@ -860,8 +863,29 @@ QtObject:
     let resultJson = responseJson["result"]
     var owners = fromJson(resultJson, CollectibleContractOwnership).owners
     owners = owners.filter(x => x.address != ZERO_ADDRESS)
-    self.tokenOwnersCache[(chainId, contractAddress)] = owners
-    let data = CommunityTokenOwnersArgs(chainId: chainId, contractAddress: contractAddress, communityId: communityId, owners: owners)
+
+    let response = communities_backend.getCommunityMembersForWalletAddresses(communityId, chainId)
+    if response.error != nil:
+      let errorMessage = responseJson["error"].getStr
+      error "Can't get community members with addresses", errorMsg=errorMessage
+      return
+
+    let communityOwners = owners.map(proc(owner: CollectibleOwner): CommunityCollectibleOwner =
+      let ownerAddressUp = owner.address.toUpper()
```

Comment:
This needed because from backend we can receive address with mixed case: "0xfdA3b750D340FD776eF2501526c0610Bf893C91B"